### ISSUE TITLE
400 Bad Request from Heroku then crash

### ISSUE THREAD

**Issue by smcallis77** [nassir-malik/IOT-ESP8266-Google-Home#1]
Nassir, I like your solution and have been trying it out, but, I've run into a problem. I am getting a 400 error from Heroku that drops the connection. The system then crashes and restarts. I can't see what's happening at Heroku so it's difficult to debug. Here are some logs from the ESP8266 and Heroku. Could you look into it?

[ErrorLog.txt](https://github.com/nassir-malik/IOT-ESP8266-Google-Home/files/1254098/ErrorLog.txt)
[Heroku Log.txt](https://github.com/nassir-malik/IOT-ESP8266-Google-Home/files/1254099/Heroku.Log.txt)

**Comment by nicolaferrara82** [comment]
I have the same issue  "at=error code=H13 desc="Connection closed without response" method=GET" but i don't know why i get this error!!!

**Comment by nassir-malik** [comment]
Did you update the Heroku name in the sketch?

-Thanks
====================================================
Nassir

**Comment by nicolaferrara82** [comment]
Yes, sure.. i've updated all required fields but i have this issue with frequently, sometimes it works well and others not,without a valid reason, the same request once it works and one does not and the assistant crash as soon as the error appears.
